packages/core/src/main/Account.test.node.ts

```diff
@@ -17,20 +17,37 @@
  *
  */
 
-const nock = require('nock');
-const {Account} = require('@wireapp/core');
-const {PayloadBundleState, PayloadBundleType} = require('@wireapp/core/dist/conversation/');
-const {APIClient} = require('@wireapp/api-client');
-const {AuthAPI} = require('@wireapp/api-client/dist/commonjs/auth/');
-const {BackendErrorLabel, StatusCode} = require('@wireapp/api-client/dist/commonjs/http/');
-const {ClientAPI, ClientType} = require('@wireapp/api-client/dist/commonjs/client/');
-const {ConversationAPI} = require('@wireapp/api-client/dist/commonjs/conversation/');
-const {GenericMessage, Text} = require('@wireapp/protocol-messaging');
-const {MemoryEngine} = require('@wireapp/store-engine');
-const {NotificationAPI} = require('@wireapp/api-client/dist/commonjs/notification/');
-const {ValidationUtil} = require('@wireapp/commons');
-const {WebSocketTopic} = require('@wireapp/api-client/dist/commonjs/tcp/');
-const {Server: MockSocketServer} = require('mock-socket');
+import {APIClient} from '@wireapp/api-client';
+import {AuthAPI, Context} from '@wireapp/api-client/dist/commonjs/auth';
+import {ClientAPI, ClientType} from '@wireapp/api-client/dist/commonjs/client';
+import {Connection, ConnectionStatus} from '@wireapp/api-client/dist/commonjs/connection';
+import {
+  ConversationAPI,
+  ConversationMessageTimerUpdate,
+  MemberJoin,
+  Rename,
+  Typing,
+} from '@wireapp/api-client/dist/commonjs/conversation';
+import {
+  CONVERSATION_EVENT,
+  CONVERSATION_TYPING,
+  ConversationMemberJoinEvent,
+  ConversationMessageTimerUpdateEvent,
+  ConversationRenameEvent,
+  ConversationTypingEvent,
+  USER_EVENT,
+  UserConnectionEvent,
+} from '@wireapp/api-client/dist/commonjs/event';
+import {BackendErrorLabel, StatusCode} from '@wireapp/api-client/dist/commonjs/http';
+import {NotificationAPI} from '@wireapp/api-client/dist/commonjs/notification';
+import {WebSocketTopic} from '@wireapp/api-client/dist/commonjs/tcp';
+import {ValidationUtil} from '@wireapp/commons';
+import {GenericMessage, Text} from '@wireapp/protocol-messaging';
+import {MemoryEngine} from '@wireapp/store-engine';
+import {Server as MockSocketServer} from 'mock-socket';
+import nock = require('nock');
```

Comment:
Can we use a proper import for nock by now?
https://github.com/nock/nock/issues/1670

Reply:
[Nope](https://github.com/nock/nock/blob/master/types/index.d.ts#L8) (at least not without `esModuleInterop: true`)

Reply:
aww :/ pity